test/unit-test/core_sntp_client_utest.c

```diff
@@ -249,6 +249,122 @@ SntpStatus_t validateServerAuth( SntpAuthContext_t * pContext,
     return validateServerAuthRetCode;
 }
 
+/* Enumeration for the API functions of the SNTP Client layer. */
+enum SntpClientApiType
+{
+    ApiInvalid,
+    ApiSendTimeRequest,
+    ApiReceiveTimeResponse
+};
```

Comment:
Nit: typedef might be useful, not in C++ though lol

Reply:
Took some liberty in the test code to avoid use of `typedef` 😛